### PR TITLE
feat(utility): add implementation about "open_api_do"

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

File: graviti/client/request.py [hunk 1]
```diff
@@ -7,15 +7,19 @@
...
 
 logger = logging.getLogger(__name__)
+PARTIAL_URL = "gatewayv2/tensorbay-open-api/v1"
 
 
-def open_api_do(
-    url: str, access_key: str, method: str, **kwargs: Any  # pylint: disable=unused-argument
-) -> Response:
+def open_api_do(url: str, access_key: str, method: str, **kwargs: Any) -> Response:
     """Send a request to the TensorBay Open API.
```
Comment (linjiX) [inline graviti/client/request.py]: `OpenAPI` without space between `Open` and `API`
Do not use `tensorbay` anymore

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/client/request.py [hunk 2]
```diff
@@ -3,31 +3,48 @@
...
 from typing import Any
+from uuid import uuid4
 
 from requests.models import Response
+from tensorbay.exception import ResponseError, ResponseErrorDistributor
+from tensorbay.utility import config, get_session
+
+from graviti.__version__ import __version__
 
 logger = logging.getLogger(__name__)
```
Comment (linjiX) [inline graviti/client/request.py:18]: `logger` seems useless

**Review by linjiX — APPROVED** [review]

File: graviti/utility/__init__.py [hunk 3]
```diff
@@ -3,3 +3,7 @@
 # Copyright 2022 Graviti. Licensed under MIT License.
 #
 """Utility module."""
+
```
Comment (linjiX) [inline graviti/utility/__init__.py:6]: Commit message:
- add quotes to `open_api_do`?
- add ".py" for "request"?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by AChenQ — APPROVED** [review]

**Review by linjiX — APPROVED** [review]

**Review by Lee-000 — APPROVED** [review]

File: graviti/utility/request.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""The basic concepts and methods of the Graviti OpenAPI."""
+
+from typing import Any
+from uuid import uuid4
+
+from requests.models import Response
```
Comment (linjiX) [inline graviti/utility/request.py:11]: shall we add requests to `requirment.txt`?

**Pull request merged**